provider/pkg/provider/provider.go

```diff
@@ -1297,20 +1297,18 @@ func (k *kubeProvider) Check(ctx context.Context, req *pulumirpc.CheckRequest) (
 		// a resource that was created out-of-band. In this case, we do not add the `managed-by` label here, as doing
```

Comment:
Is the comment here accurate?

Reply:
Partly. I'll update it to include information about SSA as well.